featurebyte/models/feature.py

```diff
@@ -223,6 +223,17 @@ def node(self) -> Node:
 
         return self.graph.get_node_by_name(self.node_name)
 
+    @property
+    def name_version(self) -> str:
```

Comment:
nit, can also consider naming this as `versioned_name` since it's more a name than a version.